shared/samples/Datadog.DynamicDiagnosticSourceBindings.Demo/LateLoadDS.NetFx/Datadog.DynamicDiagnosticSourceBindings.Demo.LateLoadDS.NetFx.csproj

```diff
@@ -54,7 +54,7 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <PropertyGroup>
-    <SharedSrcBaseDir>$(MSBuildThisFileDirectory)..\..\..\src\managed-src</SharedSrcBaseDir>
+    <SharedSrcBaseDir>$(EnlistmentRoot)\shared\src\managed-src</SharedSrcBaseDir>

```

Comment:
Could `SharedSrcBaseDir` be defined once in the file `dd-trace-dotnet/shared/Directory.Build.props` instead of in all of the various projects? Or would that be disruptive to other project files in the build?

Reply:
This works fine as-is, I was just hoping to define it once and minimize future copy/paste errors 😄

Reply:
I think, it's a good idea. I'd prefer to do this in a future PR once the profiler has been merged into this repo. At that time, we would place that definition high enough in the folder structure for everyone to be able to access it.